diff --git a/Cargo.toml b/Cargo.toml
deleted file mode 100644
index f7aa5cb..0000000
--- a/Cargo.toml
+++ /dev/null
@@ -1,2 +0,0 @@
-[workspace]
-members = ["bpf-sys", "redbpf", "redbpf-probes", "redbpf-macros", "cargo-bpf", "redbpf-tools"]
diff --git a/bpf-sys/src/headers.rs b/bpf-sys/src/headers.rs
index 841fa71..f744079 100644
--- a/bpf-sys/src/headers.rs
+++ b/bpf-sys/src/headers.rs
@@ -102,7 +102,7 @@ fn kernel_headers_path() -> Result<KernelHeaders, HeadersError> {
         let path = PathBuf::from(s);
         KernelHeaders {
             source: path.clone(),
-            build: path
+            build: path.parent().unwrap().join("build"),
         }
     })
     .or_else(lib_modules_kernel_headers)
diff --git a/cargo-bpf/src/build_constants.rs b/cargo-bpf/src/build_constants.rs
index f1e01c3..597ee47 100644
--- a/cargo-bpf/src/build_constants.rs
+++ b/cargo-bpf/src/build_constants.rs
@@ -32,7 +32,7 @@ lazy_static! {
             "-Wall",
             "-Werror",
             "-Wunused",
-            "-Wno-frame-address",
+            //"-Wno-frame-address",
             "-Wno-unused-value",
             "-Wno-pointer-sign",
             "-Wno-compare-distinct-pointer-types",
